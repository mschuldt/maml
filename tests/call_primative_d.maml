arduino = Arduino(desktop=True)

@block
def test():
    print_i(5)
    print_i(5 + 6)
    print_i(1 + 2 + 3)
    test_add2(1, 2)
    test_add3(1+1, 2, 3)
    test_add4(1+1, 2, 3, 4)
    test_add5(1+1, 2, 3, 4, 5)
    test_add6(1+1, 2, 3, 4, 5, 6)
    die(0)
    
arduino.send(test)
