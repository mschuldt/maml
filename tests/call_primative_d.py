arduino = Arduino(desktop=True)

@block
def test():
    print_i(5)
    print_i(5 + 6)
    print_i(1 + 2 + 3)
    quit(0)
    
arduino.send(test)
