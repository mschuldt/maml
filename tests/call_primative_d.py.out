Initializing avm...
Ready.

ERROR: unrecognized bytecode: '23'
==> 5
==> 11
==> 6
bye.
